index.html

```diff
@@ -23,7 +25,85 @@
 		</main>
 
 		<!-- aside -->
-		<aside id="aside"></aside>
+		<aside id="aside">
+
+			<!-- status -->
+			<div class="status">
+				<div class="profile">
+					<div class="profile__img">
+						<img src="./aside_img/user.jpg" alt="user">
+					</div>
+					<div class="profile__info">
+						<h3>Mon pseudo</h3>
+						<p>il y'a 10 minutes</p>
+					</div>
+				</div>
+				<div class="status__info">
+					<p>
+						OH right. I forgot about the battle.
+						i just told you! You've killed me! You don't know how to do any of those.
+						Hey, you add a one and two zeros to that or we walk! Don't jump!
+					</p>
+				</div>
+			</div>
+
+			<!-- suggestion -->
+			<div class="suggestion">
+				<h2>Suggestion</h2>
+				<hr>
```

Comment:
Est ce que tu peux enlever le hr et mettre un border-bottom a h2 ?

Reply:
d'accord je vais le faire

Reply:
Bro essaie de mettre la photo du user dans assets/images

Reply:
Il est préférable d'avoir tous les images dans un même sous dosier

Reply:
d'accord c'est noté